templates/projects/console/Program.cs

```diff
@@ -1,13 +1,10 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
```

Comment:
Why the `using` should be removed? As I understand there is a reason behind most common imports being used in blank templates:
https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/ConsoleApp/Program.cs